src/js/components/DataChart/utils.js

```diff
@@ -96,7 +102,7 @@ export const createDateFormat = (firstValue, lastValue, full) => {
     else if (delta < 86400000)
       // less than 1 day
       options = { hour: 'numeric' };
-    else if (delta < 2592000000)
+    else if (delta < 2592000000 || delta === 0)
```

Comment:
```suggestion
    else if (delta < 2592000000)
```
Let's revert this change as well